RELEASE_PROCESS.md

```diff
@@ -27,10 +27,7 @@ development efforts occur on the *main* branch.  Branches with a
 
 # Prechecks
 
-Two days before actually cutting a release (including RCs), send an announcement to the
-[podman-desktop](mailto:podman-desktop@lists.podman.io)
-mailing list about the upcoming release. This will help the Podman Desktop team test and schedule
-their own new release.
+Release timing, including RC’s, should be documented on our Podman release calendar, so other projects will know to test and prepare for our releases.
```

Comment:
can we link the podman release calendar ?

Reply:
This should honestly be removed entirely. We have an internal calendar for this, but upstream releases are 2nd week of February, May, August, November.

Reply:
Saying that should be sufficient.